touchforms/formplayer/templates/fullform.html

```diff
@@ -3,10 +3,11 @@
   <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
-    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/jquery-1.8.3.min.js"></script>
-    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/jquery-ui.min.js"></script>
+    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/jquery-1.8.3.min.js"></script>
+    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/jquery-ui.min.js"></script>
     <link rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.5/themes/ui-lightness/jquery-ui.css">
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/shortcut.js"></script>
+    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/underscore.js"></script>
```

Comment:
so i'm a bit conflicted about this, our HQ does load underscore.js, but i'd like to be able to test the js code in touchforms without having to depend on HQ. maybe a better solution would be to check if underscore has been loaded first though it's unclear if we can do that in the template